addon/components/lazy-render-wrapper.js

```diff
@@ -0,0 +1,149 @@
+import Ember from 'ember';
+
+const { computed, get, run, $ } = Ember;
+
+// https://github.com/emberjs/rfcs/issues/168
+export default function getParent(view) {
+  if (get(view, 'tagName') === '') {
+    // Beware: use of private API! :(
+    if (Ember.ViewUtils && Ember.ViewUtils.getViewBounds) {
+      return $(Ember.ViewUtils.getViewBounds(view).parentElement);
+    } else {
+      return $(view._renderNode.contextualElement);
```

Comment:
Actually, it looks like [the mentioned issue](https://github.com/emberjs/rfcs/issues/168) answers that question. Could you also link to https://github.com/emberjs/ember.js/pull/12500 in the code comment here?